test/docker-compose.yml

```diff
@@ -3,31 +3,43 @@ services:
   selenium-hub:
     image: ${DOCKER_REGISTRY}selenium/hub:3.141.59
     container_name: selenium-hub
+    environment: 
+      - NODE_MAX_SESSION=3
+      - NODE_MAX_INSTANCES=3
   selenium-chrome:
-    image: ${DOCKER_REGISTRY}selenium/node-chrome:3.141.59
+    image: ${DOCKER_REGISTRY}selenium/node-chrome-debug:3.141.59
```

Comment:
debug images gebruiken heeft weinig zin als de VNC poort niet exposed word. Gezien we de containers automatisch schalen, is het voorlopig onmogelijk om deze VNC poorten te exposen omdat er dan x-aantal containers proberen te binden op poort y.

Reply:
Aangepast, maar ik zie toch nog wat verschillen met de blueprint. Misschien nog eens verder bekijken wat de verschillen zijn?